com.francescogaglione.chronos.json

```diff
@@ -0,0 +1,46 @@
+{
+  "id": "com.francescogaglione.chronos",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "chronos",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```

Comment:
you have to ask an exception for this.

Reply:
error code is `finish-args-unnecessary-xdg-config-cosmic-ro-access`

Reply:
@bbhtt 
https://github.com/flathub-infra/flatpak-builder-lint/pull/490
Done it